src/js/shared/application-form.js

```diff
@@ -137,18 +136,18 @@ exports.createApplicationForm = function createApplicationForm(validateFile = tr
     }),
     team_apply: fields.boolean({
       label: 'Are you applying as part of a team?',
-      note: 'We will not process your application until you have been entered into a team via the application form (after submitting this form).',
-      widget: checkboxWidget('Yes, I am applying as part of a team. One of our members will fill out the team application form.'),
+      note: 'We won’t process your application until you’ve been entered into a team using the application form, after submitting this form.',
```

Comment:
How about "We won’t process your application until you’ve been entered into a team using the team application form.  The team application form can be submitted after every team member has submitted this form."?